#! /bin/sh /usr/share/dpatch/dpatch-run
## 28_check_pgsql_include_for_8.3.dpatch by Jan Wagner <waja@cyconet.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: include pg_config_manual.h since NAMEDATALEN definition moved there with psql 8.3 

@DPATCH@
diff -urNp -x '*.orig' nagios-plugins-2.3.3.org/plugins/check_pgsql.c nagios-plugins-2.3.3/plugins/check_pgsql.c
--- nagios-plugins-2.3.3.org/plugins/check_pgsql.c	2019-12-04 22:53:08.000000000 +0100
+++ nagios-plugins-2.3.3/plugins/check_pgsql.c	2021-04-19 12:01:17.181840152 +0200
@@ -36,6 +36,7 @@ const char *email = "devel@nagios-plugin
 #include "utils.h"
 
 #include "netutils.h"
+#include "pg_config_manual.h"
 #include <libpq-fe.h>
 #include <pg_config_manual.h>
 
